content/en/docs/tutorials/services/pods-and-endpoint-termination-flow.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Pods And Endpoints Termination Flow
+content_type: tutorial
+weight: 60
+---
+
+
+<!-- overview -->
+
+## The Kubernetes model for connecting containers
+
+Now that you have a continuously running, replicated application, that is
+exposed on a network, let's look at Pods termination flow and ways to implement
+a graceful connections draining.
+
+There are often a need to terminate a Pod - be it for upgrade or scale down.
+In order to improve application availability, it may be important to implement
+a proper active connections draining. This tutorial explains the flow of
+Pod termination in connection with the corresponding endpoint state and removal.
+
+This tutorial uses a simple nginx web server to demonstrate the concept.
```

Comment:
Can we not append this to the previous sentence as - This tutorial explains the flow of
Pod termination in connection with the corresponding endpoint state and removal by using a simple nginx web server to demonstrate the concept?